src/gui/maptools/qgsmaptoolcapture.cpp

```diff
@@ -642,10 +642,6 @@ int QgsMapToolCapture::fetchLayerPoint( const QgsPointLocator::Match &match, Qgs
   {
     if ( ( match.hasVertex() || match.hasLineEndpoint() ) )
     {
-      if ( sourceLayer->crs() != vlayer->crs() )
```

Comment:
Can you update the documentation for fetchLayerPoint? It explicitly mentions "if if has the same CRS as the current layer", and "1 if not applicable (CRS mismatch / invalid layer)"

Reply:
Right! done